app/trip/[id]/edit/page.tsx

```diff
@@ -0,0 +1,34 @@
+'use client'
+
+import { Box, Container, Heading, useColorModeValue } from '@chakra-ui/react'
+// import { Loading } from '@/components/loading'
```

Comment:
I'd say it's better to delete comment out source, cuz it's unnecessary.

Reply:
I will use it next logic.. Can I just leave it? 🙏